envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/HostHeaderRewritingTest.kt

```diff
@@ -11,16 +11,18 @@ internal class HostHeaderRewritingTest : EnvoyControlTestConfiguration() {
 
     companion object {
         const val customHostHeader = "x-destination-host"
+        val httpBinContainer: HttpBinContainer = HttpBinContainer()
 
         @JvmStatic
         @BeforeAll
         fun setupTest() {
+            httpBinContainer.start()
```

Comment:
We should also stop container in @AfterAll method. Look at the EnvoySANValidationTest test for reference

Reply:
added container cleanup